js/js-functions_basic-functions/js/index.js

```diff
@@ -36,6 +66,16 @@ Then log all variables to the console again.
  - Now increase the number of sales 2 more times and log the data after each increase.
 */
 
-// --v-- write your code here --v--
+function logBookData() {
+  console.log(bookTitle, "\n", author, "\n", bookRating, "\n", bookSales);
+}
+
+logBookData();
+
+bookSales = 13;
+
+logBookData();
+
+bookSales = 27;
```

Comment:
In my RunJS it logs the reassigned variable another time outside the output of the function. I've seen this also in my code and in other solutions. I wonder how to avoid that.

Reply:
Hey @TS-Severin, I don't understand what you mean that it log outside the output. It looks fine on my computer. Can you attach a screenshot maybe?

Reply:
![function](https://github.com/yuvaliacob/challenges/assets/127843093/ca1f449b-d717-4410-a5dd-6fb162c1cd76)